packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -1467,52 +1468,86 @@ export default abstract class SqlIntegration
 
     return { tableData: results.rows };
   }
-  getSchemaFormatConfig(schemaFormat: SchemaFormat): SchemaFormatConfig {
+  getSchemaFormatConfig(schemaFormat: AutoMetricSchemas): SchemaFormatConfig {
     switch (schemaFormat) {
+      case "amplitude": {
+        return {
+          trackedEventTableName: `EVENTS_${
+            this.settings.schemaOptions?.projectId || `*`
+          }`,
+          eventColumn: "event_type",
+          timestampColumn: "event_time",
+          userIdColumn: "user_id",
+          anonymousIdColumn: "amplitude_id",
+          getMetricTableName: () =>
+            this.generateTablePath(
+              `EVENTS_${this.settings.schemaOptions?.projectId || `*`}`
```

Comment:
We shouldn't ever actually wildcard this, as we have a check on the frontend that throws an error if the projectId isn't set, but keeping this.